index.html

```diff
@@ -1162,25 +1156,24 @@ <h4>
   <section> <h3>Save and restore game progress with another device</h3>
     <p>
       Filtering of relevant data sources can be done by the use of
-      the <a>NDEFScanOptions</a>. Below we accept the record identifier URL with
-      "`/mypath/mygame/`" in its path from "`mygame.com`"
-      domain and its subdomains. When we read the data, we immediately update
-      the game progress by issuing a push with a custom NDEF data layout.
+      the <a>NDEFScanOptions</a>. Below we use a custom record identifier so
+      that when we read the data, we immediately update the game progress by
+      issuing a push with a custom NDEF data layout.
     </p>
     <p>
       The example allows reading and pushing to both peers and tags,
       whichever one is tapped first.
     </p>
     <pre class="example">
       const reader = new NDEFReader();
-      await reader.scan({ id: "https://mygame.com/mypath/mygame" });
+      await reader.scan({ id: "my-game-progress" });
```

Comment:
according to NDEF spec ids are supposed to be urls

Reply:
![Screenshot 2019-11-25 at 09 31 24](https://user-images.githubusercontent.com/1101339/69524473-6abbe100-0f66-11ea-869c-4891a9853ca8.png)

Reply:
semanticly it is an url (can be relative so you can say the examples are) but we should at least note that in a note or something

Reply:
Right, I forgot about that.

Reply:
I've edited example description at https://github.com/w3c/web-nfc/pull/446/commits/7c2d2c10fb5cfa6e4f2606d56544ecd91bc68373

Reply:
Where we introduce the id field, are we making clear that it is a url?

Reply:
updated example looks good

Reply:
> Where we introduce the id field, are we making clear that it is a url?

I've updated `id` introduction at https://github.com/w3c/web-nfc/pull/446/commits/a78d5fbea5713d7521c7ba15f3560af124468575

Reply:
If we include relative URLs, what they are relative to, and how does the implementation add the base URL when the record is written?

Reply:
The implementation does not add any base URL but just writes the relative URL as is.